pkgs/applications/misc/k40-whisperer/default.nix

```diff
@@ -46,14 +48,29 @@ in stdenv.mkDerivation rec {
 
     mkdir -p $out/bin
     mkdir -p $out/lib/udev/rules.d
+    mkdir -p $out/share/icons/hicolor/128x128/apps
+    mkdir -p "$out/share/applications"
 
     ln -s ${udevRule} $out/lib/udev/rules.d/97-k40-whisperer.rules
+    ln -s "$desktopItem/share/applications/k40whisperer.desktop" "$out/share/applications/k40whisperer.desktop"
```

Comment:
Referencing variables before using this is weird, this confused me for a couple of minutes